lua/lvim/core/telescope.lua

```diff
@@ -25,7 +25,16 @@ function M.config()
       layout_config = {
         width = 0.75,
         preview_cutoff = 120,
-        horizontal = { mirror = false },
+        prompt_position = "bottom",
```

Comment:
What was the reason behind this? It breaks other layouts, e.g. `<leader>Lf`

```suggestion
```

Reply:
shit this was not supposed to be in here 😅